src/components/plugins-list/style.module.css

```diff
@@ -97,6 +89,14 @@ ul.pluginsList li p {
   background-color: #00805b;
 }
 
+.badgePill.badgeCommunity {
+  background-color: #737373;
+}
+
+.badgePill.badgeDeprecated {
+  background-color: #870b2a;
+}
```

Comment:
I'm not the styling expert, but I wonder if the Community background could be made less bland and the Deprecated background less prominent?

This has to work for both light and dark themes.

Reply:
Just pushed up another change, thoughts on these instead?

![image](https://github.com/cypress-io/cypress-documentation/assets/22203690/c4602b0b-62c3-4e3b-bd3f-d3a380c25b7d)